docs/user-docs/experience-blocks-analytics/README.md

```diff
@@ -0,0 +1,38 @@
+
+## Creating personalized content
+
+To get started, create a Personalized Content block (aka XB) from the list of available blocks.
+
+![adding experience block](../assets/create-xb.gif)
+
+Add some fallback content. This content will be displayed by default if none of the specific audience variants apply to a user.
+
+Next you'll want to create some personalized content to make the user's experience unique to the audience segment you are trying to reach. You create a variant by clicking the + icon. You will be prompted to select an audience. If no audiences exist, or you want to use a new audience, you can use the "Add New" button to create a new audience. Otherwise you can select an existing audience and add your personalized content for that segment of users.
+
+![Adding an audience](../assets/create-audience-xb.gif)
+
+## Creating audiences
+
+You can create new audiences within the content editing experience or from the Audiences page in the admin. In either case, the process is the same. Within the New Audience screen, you can give your audience a name, and set the variables in the dropdowns to determine how to identify this segment of users, for example browser or geographic region. Once the audience is set, you can use that to deliver unique content based on the parameters you've set. Before you can use the audience, switch the audience to Active on the right hand side. Now the audience is ready to be used in your personalized content!
+
+![Personalized content](../assets/personalized-content.gif)
+
+
+## Add conversion goals to measure engagement
+
+There are three ways that you can track conversions built into Altis Experience Blocks. These are "Impressions", "Click on any link" and "Submit a form". By default, impressions will be selected, and the statistics that display will be based on that data.
+
+![Impressions](../assets/conversions-impressions.png)
+
+However, you can change the conversion goal to any of the available options and the goal tracking will reflect the new goal after the post is updated. Any action of the type selected (clicking a link or submitting a form) will mark a conversion that will be tracked for that variant.
+
+![Update the conversion goal](../assets/set-cta-goal.gif)
+
+## Measuring performance
+
+Once you have added personalized content and audiences to your post and have published them, you will be able to see the number of XB views per variant in the sidebar on the right when you click on the Personalized Content block. If you added a conversion goal to your personalized content blocks, you will be able to see the percentage conversion against the total number of views for each block. To compare how well your personalized content is performing, you must have a fallback variant to compare the personalized content metrics against.
```

Comment:
```suggestion
Once you have added personalized content and audiences to your post and have published them, you will be able to see the number of XB views per variant in the sidebar on the right when you click on the Personalized Content block. If you added a conversion goal to your personalized content blocks, you will be able to see the percentage conversion against the total number of views for each variant. To compare how well your personalized content is performing, you must have a fallback variant to compare the personalized content metrics against.
```

The last sentence isn't something we have just yet but I think is the intention. You don't actually have to have any content in the fallback. Maybe we should mention that specifically in the opening section.

Reply:
just omitting the last sentence, we can add it back later